### PR TITLE
feat: Create login page.

### PR DESCRIPTION
**Pull request by mllrdev-glenn-henares**

### CODE REVIEW

File: src/data/User.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+interface Name {
```
Comment (mllrdev-carlo-medel) [inline src/data/User.ts]: In practice prefix the name of the interface with the letter "I" to indicate that it's an interface. 
```suggestion
interface IName {
```

File: src/data/User.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    firstName: string;
+    middleName: string;
+    lastName: string;
+}
+
+interface User extends Name {
```
Comment (mllrdev-carlo-medel) [inline src/data/User.ts]: See comment above.

File: src/theme/core.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+app {
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    color: #000000;
+}
+
+ion-button {
+    height: 0.3in;
+    width: 1in;
+    --background-color: --ion-color-medium-contrast;
+}
```
Comment (mllrdev-carlo-medel) [inline src/theme/core.css]: Is this global design necessary? I think you can just do this in component level?

File: src/router/index.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { createRouter, createWebHistory } from '@ionic/vue-router';
 import { RouteRecordRaw } from 'vue-router';
-import Home from '../views/Home.vue'
+import Home from '../pages/Home.vue'
+import Login from '../pages/Login.vue'
```
Comment (mllrdev-carlo-medel) [inline src/router/index.ts]: Let's use the complete path when importing. Since we want to avoid this scenario e.g. `../../../../pages/Home.vue`

**Review by mllrdev-carlo-medel — CHANGES_REQUESTED** [review]

**Review by KarlZablan — CHANGES_REQUESTED** [review]
Please use kebab case when naming your files.

File: src/router/index.ts [hunk 5]
```diff
@@ -11,6 +12,11 @@ const routes: Array<RouteRecordRaw> = [
     path: '/home',
     name: 'Home',
     component: Home
+  },
+  {
+    path: '/login',
+    name:'Login',
```
Comment (KarlZablan) [inline src/router/index.ts]: please create an enum for the route names.

**Review by KarlZablan — CHANGES_REQUESTED** [review]

File: src/theme/core.css (hunk 3 above)
Comment (mllrdev-glenn-henares) [reply src/theme/core.css]: It's intended for generic designs like colors and fonts. Removed height and width. Positions and size will be component side.

File: src/router/index.ts (hunk 4 above)
Comment (mllrdev-glenn-henares) [reply src/router/index.ts]: Changed.

File: src/router/index.ts (hunk 5 above)
Comment (mllrdev-glenn-henares) [reply src/router/index.ts]: Enum file created.

**Review by mllrdev-carlo-medel — APPROVED** [review]

**Pull request merged**